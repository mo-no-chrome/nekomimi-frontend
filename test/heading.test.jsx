import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../src/js/heading';

describe('<Header>', () => {
  test('renders correctly', () => {
    const name = 'Nekomimi';
    const props = { name };
    const tree = renderer.create(<Header {...props} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
