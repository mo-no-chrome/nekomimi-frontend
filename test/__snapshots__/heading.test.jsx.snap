// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders correctly 1`] = `
<h2>
  Hello 
  Nekomimi
   World!
</h2>
`;
